src/installing-workflow/configuring-object-storage.md

```diff
@@ -11,26 +11,18 @@ A variety of Deis components rely on an object storage system to do their work.
 
 These components are flexible and can work out of the box with almost any system that is compatible with the [S3 API](http://docs.aws.amazon.com/AmazonS3/latest/API/APIRest.html).
 
-Note: object storage configuration has not been standardized across all components in our beta release. As such, configuration instructions differ for each component. We plan to remediate this problem in our next release. Please see [deis/deis#4966](https://github.com/deis/deis/issues/4966) for our progress on that work.
-
 ## Minio
 
 Additionally, Deis ships with a [Minio](http://minio.io) [component](https://github.com/deis/minio) by default, which provides in-cluster, ephemeral object storage. This means that _if the Minio server crashes, all data will be lost_. Therefore, **Minio should be used for development or testing only**.
 
 In our beta release, the components listed above are configured by default to automatically use the Minio [service][k8s-service] for object storage.
 
-## Google Cloud Storage
-
-[Google Cloud Storage](https://cloud.google.com/storage/) (GCS) can interoperate with the S3 API using a feature called [interoperability](https://cloud.google.com/storage/docs/interoperability). If you choose to use GCS for object storage, you'll have to turn on this interoperability mode. In order to do so, please follow the steps in the [GCS migration documentation](https://cloud.google.com/storage/docs/migrating?hl=en_US#migration-simple).
-
-We recommend storing these and all other credentials as Kubernetes secrets. See the below sections for details on configuring each component for details.
-
 # Configuring the Deis Components
 
 Every Deis component that relies on object storage relies on the following two inputs for configuration:
 
-- One or more environment variables that describe what object storage system to use
-- One or more files to provide access credentials for the object storage system.
+- An environment variable that describe what object storage system to use.
+- A configuration file ([objectstorage.toml][objectstorage-toml]) to provide access credentials for the object storage system.
 	- We suggest storing the contents of these files in [Kubernetes secrets][k8s-secret] and mounting them as volumes to each pod
```

Comment:
For consistency with the other bullet points in this section, can you put a `.` character at the end of this bullet point?